microcosm_postgres/operations.py

```diff
@@ -29,6 +49,20 @@ def drop_all(graph):
 
     """
     Model.metadata.drop_all(graph.postgres)
+    drop_alembic_table(graph)
```

Comment:
No, because I don't have a similar boolean from the previous line to `OR` over. It's not a meaningful return value unless it captures whether *anything* was dropped.

(I added the return value primarily because it helped me test.)